documentation/configuringDynawo/DynawoInputFiles.tex

```diff
@@ -679,19 +679,32 @@ \subsubsection{Define sets of static references}
 
 \subsection{par file}
 
-\subsubsection{Set a reference to a value stored in the iidm file}
+\subsubsection{Set a reference to a value stored in the iidm file or a given par file}
 \label{sec:Set a reference to a value stored in the iidm file}
 
 If the parameter's value is stored in the iidm file, a reference to this value can be added in the par file as in the example bellow:
-\begin{lstlisting}[language=XML, morekeywords={reference}]
+\begin{lstlisting}[language=XML, morekeywords={iidmreference}]
```

Comment:
Why this change?

Reply:
Because I added another lstlisting below (about PAR reference)

Reply:
I think this is just to tell LaTeX that reference should be seen as a xml key word

Reply:
Ok so I put "reference" back